background/web3.js

```diff
@@ -17,6 +17,8 @@
 import Config, { DEFAULT_CONFIG } from './config';
 import { TRANSPORT_UNINITIALIZED } from '../shared';
 
+const FAILURE = 'Failed to fetch';
```

Comment:
Just looks safer (if an OS adds special characters, if we want to port to Firefox where to message is somehow the same but different casing, wording, etc.)